src/query-builder/update-query-builder.ts

```diff
@@ -594,6 +594,69 @@ export class UpdateQueryBuilder<DB, UT extends keyof DB, TB extends keyof DB, O>
     })
   }
 
+  /**
+   * This can be used to add any additional SQL to the front of the query __after__ the `update` keyword.
+   *
+   * ### Examples
+   *
+   * ```ts
+   * await db.updateTable('person')
+   * .modifyFront(sql.raw('-- This is a comment\n'))
+   * .set({ age: 39 })
+   * .where('first_name', '=', 'John')
+   * .execute()
```

Comment:
Is there a real-world use case for this?

Reply:
No, made just for consistency, will remove it